denizen_scripts/survival/repo-link/meeseeks_shit/Meeseeks.dsc

```diff
@@ -439,11 +439,11 @@ Meeseeks_Shop:
                             - define LockedChests <[LockedChests].with[<[Chest].simple>].as[<list[<npc.id>|<[UUID]>]>]>
                             - adjust <[Chest]> lock:<&b>K<&3>ey<[UUID].to_secret_colors>
                             - if <npc.has_flag[ShopOpen]>:
-                                - if !<npc.flag[ItemPrices].as_map.list_keys.contains[<[Chest].inventory.list_contents.filter[material.name.is[!=].to[Air]].parse[with[quantity=1]].deduplicate>]>:
+                                - if !<npc.flag[ItemPrices].as_map.keys.contains[<[Chest].inventory.list_contents.filter[material.name.is[!=].to[Air]].parse[with[quantity=1]].deduplicate>]>:
                                     - narrate format:Colorize_Yellow "New shop items found! Shop closed until price verified."
                                     - flag npc ShopOpen:!
                         - flag server Meeseeks.LockedChests:<[LockedChests]>
-                        - run Meeseeks_Shop def:<list_single[<list[menu/Meeseeks_Shop|action/KeyBox|npc/<npc>]>]>
+                        - run Meeseeks_Shop def:<list_single[menu/Meeseeks_Shop|action/KeyBox|npc/<npc>]>
```

Comment:
`list_single` is made to create a list with a single entry, which is useful for passing lists within lists, but passing a list this way inside `list_single` puts it in a single list as an element to the receiving script, the list is needed here